umap/views.py

```diff
@@ -190,9 +190,9 @@ def get_context_data(self, **kwargs):
         q = self.request.GET.get("q")
         results = []
         if q:
-            where = "to_tsvector(name) @@ plainto_tsquery(%s)"
+            where = "to_tsvector(name) @@ websearch_to_tsquery(%s)"
             if getattr(settings, "UMAP_USE_UNACCENT", False):
-                where = "to_tsvector(unaccent(name)) @@ plainto_tsquery(unaccent(%s))"  # noqa
+                where = "to_tsvector(unaccent(name)) @@ websearch_to_tsquery(unaccent(%s))"  # noqa
```

Comment:
```suggestion
                where = "to_tsvector('umapdict',name) @@ websearch_to_tsquery('umapdict',%s)"  # noqa
```
You no longer need unaccent (case has been removed thanks to your FTS config) and  you must specify FTS config.